docs/docs/cloud/cloud-cli.mdx

```diff
@@ -48,6 +48,80 @@ meltano cloud config env delete TAP_GITHUB_AUTH_TOKEN
 
 See the [reserved variables](/cloud/platform#reserved-variables) docs for more details on variables that are reserved for use by Meltano Cloud.
 
+### `notification`
+
+The `notification` command provides a way for managing your [Meltano Cloud notifications](concepts#meltano-cloud-notifications).
+
+#### `list`
+
+List all notifications configured for your current default project or organization.
+
+```sh
+meltano cloud config notification list
+```
+
+#### `set`
+
+Set a new notification or override an existing one.
+
+Interactive mode
+```sh
+meltano cloud config notification set <type>
+```
+
+Non-interactive mode:
+```sh
+meltano cloud config notification set <type> --recipient <target_recipient> --filter <filter>
+````
+
+**`options`**
+
+
+`--recipient` (Required): The recipient to send the notification to.
+
+`--filter`: An optional filter you want to check against, you can supply multiple filters which will all be joined as an OR expression.
+Values within the filter are validated as an AND Expression.
+
+```sh
+meltano cloud config notification set webhook --recipient http://test+melty --filter '{"events": ["job_created", "job_succeeded"], "status": ["succeeded"]}' --filter '{ "status": ["failed"]}'
```

Comment:
@leoroese presumably all of these are case-sensitive, correct? Can we add a line indicating as much?

Reply:
Oh yeah good call